src/main/java/frc/robot/Robot.java

```diff
@@ -22,7 +25,16 @@
   private String m_autoSelected;
   private final SendableChooser<String> m_chooser = new SendableChooser<>();
 
-  private Drive drivetrain;
+  private PWMSparkMax leftMotorController1; 
+  private PWMSparkMax leftMotorController2; 
+  private SpeedControllerGroup leftSide; 
+
+  private PWMSparkMax rightMotorController1;
+  private PWMSparkMax rightMotorController2;
+  private SpeedControllerGroup rightSide;
```

Comment:
The Robot.java doesn't need to actually have fields to hold reference to the Sparks and SpeedControllerGroups - those can just be instantiated right when we pass them in to the DriveImpl constructor, either inline or by declaring a local variable within the method where we call DriveImpl() and passing that variable in.